spring-native-configuration/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationHints.java

```diff
@@ -1,27 +0,0 @@
-/*
- * Copyright 2019-2021 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.springframework.boot.autoconfigure.integration;
-
-import org.springframework.nativex.type.NativeConfiguration;
-import org.springframework.nativex.hint.NativeHint;
-import org.springframework.nativex.hint.ResourceHint;
-
-@NativeHint(trigger = IntegrationAutoConfiguration.class,
-	resources = @ResourceHint(patterns="META-INF/spring.integration.default.properties")
-)
-public class IntegrationHints implements NativeConfiguration {
```

Comment:
Removed this class since there is no such a `META-INF/spring.integration.default.properties` resource in SI starting with version `5.5` any more